cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,25 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://conduit.mate.academy/user/login');
+  });
+
   it('should provide an ability to log in', () => {
+    cy.url()
+      .should('include', '/login');
+  });
+
+  it('should be fill in the fields', () => {
+    cy.get(':nth-child(1) > .form-control')
+      .type('ma150269@gmail.com');
+
+    cy.get(':nth-child(2) > .form-control')
+      .type('1234');
+
+    cy.get('.btn').click();
 
+    cy.get(':nth-child(4) > .nav-link')
+      .should('contain.text', 'romaa');
```

Comment:
Ensure that the username 'romaa' is expected and correct for the test scenario. If this is dynamic, consider using a variable or a fixture.